sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentClassifierAdministrationLiveTests.cs

```diff
@@ -36,18 +36,41 @@ public async Task BuildDocumentClassifierCanAuthenticateWithTokenCredential()
         {
             var client = CreateDocumentModelAdministrationClient(useTokenCredential: true);
```

Comment:
This test was creating this client set up with AAD authentication but was not using it anywhere in code.